src/patternfly/components/EmptyState/empty-state.scss

```diff
@@ -17,40 +17,49 @@
   --#{$empty-state}--m-lg__content--MaxWidth: #{pf-size-prem(600px)};
 
   // Icon
-  --#{$empty-state}__icon--MarginBottom: var(--#{$pf-global}--spacer--lg);
-  --#{$empty-state}__icon--FontSize: var(--#{$pf-global}--icon--FontSize--xl);
-  --#{$empty-state}__icon--Color: var(--#{$pf-global}--icon--Color--light);
-  --#{$empty-state}--m-xs__icon--MarginBottom: var(--#{$pf-global}--spacer--md);
-  --#{$empty-state}--m-xl__icon--MarginBottom: var(--#{$pf-global}--spacer--xl);
+  --#{$empty-state}__icon--MarginBottom: var(--pf-t--global--spacer--lg);
```

Comment:
Could we update this to `--#{$empty-state}__content--RowGap: var(--pf-t--global--spacer--md);`

Then remove the margins from individual elements?